development/kill.start.ips.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
```

Comment:
Consider adding `set -e` so that the script exits on errors.

Reply:
thanks! fixed in 2f38225